lnd_services.go

```diff
@@ -453,6 +458,33 @@ func NewLndServices(cfg *LndServicesConfig) (*GrpcLndServices, error) {
 		log.Infof("lnd is now fully synced to its chain backend")
 	}
 
+	// If requested, wait until the chain notifier RPC is ready before we
+	// return. This ensures sub-servers relying on the notifier don't fail
+	// during startup.
+	if cfg.BlockUntilChainNotifier {
+		if !hasBuildTag(services.Version, "chainrpc") {
```

Comment:
is this referring to the tags of lndclient or is this related to the tags sof LND which the client is connected to ?

Reply:
I wonder if we really need to rely on build tags, will the waitForChainNotifier func not just error when there is no notifier ?